### PR TITLE
Data Migration Tool v1.3

### PR DESCRIPTION
**Pull request by v-nikitaraut**
## Description
Export with history and delete changes.

## Testing
1. Tested locally.
2. Tested on Azure function app
3. Tested Deep check

### CODE REVIEW

File: GeoPol.xml
Comment (erikhoward) [inline GeoPol.xml:1]: The GeoPol.xml file is usually ignored after it's been created. Is there a reson for it's inclusion in this PR?

File: incremental-copy-docs/README.md [hunk 1]
```diff
@@ -37,7 +52,9 @@ To quickly deploy the Migration tool, you can use the Azure deployment below. Pl
 
 		[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fapiforfhir-migration-tool%2Fmain%2Finfra%2Fmain.json/createUIDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fapiforfhir-migration-tool%2Fmain%2Finfra%2FuiDefForm.json)
 
-    2. Or, deploy the migration tool manually
+    **NOTE** : To avoid azure resource already deployed with the same name use unique Prefix for FHIR Migration Tool resources during deployment.
```
Comment (erikhoward) [inline incremental-copy-docs/README.md:55]: Capitalization: "azure" should be "Azure".

File: incremental-copy-docs/README.md [hunk 2]
```diff
@@ -76,6 +93,8 @@ To quickly deploy the Migration tool, you can use the Azure deployment below. Pl
 				- <*path-to-template*>: Provide the path to your ARM/Bicep template file i.e. main.json under infra folder.
 				- <*path-to-parameter*>: Specify the path to the parameters file i.e. armmain.parameters.json under infra folder.
 
+      **NOTE** : To avoid azure resource already deployed with the same name use unique Prefix for FHIR Migration Tool resources during deployment.
```
Comment (erikhoward) [inline incremental-copy-docs/README.md:96]: Capitalization: "azure" should be "Azure".

File: incremental-copy-docs/README.md [hunk 3]
```diff
@@ -169,6 +204,33 @@ Dasboard contain below details.
...
 	- Average available memory.
 
+### Table Storage Monitoring
+
+During the deployment of data migration tool , the table storage [chunk and export table] linked to Function App is used to store and monitoring the data migration from Azure API for FHIR to Azure Health Data Service FHIR service. This table gives the overview and details of each export-import runs.
+
+There are two table storage created during deployment.
+
+1. Chunk: 
+	- It store the how many run have been done or started for the migration.
```
Comment (erikhoward) [inline incremental-copy-docs/README.md:214]: Grammar issue.

File: GeoPol.xml
Comment (v-nikitaraut) [reply GeoPol.xml:1]: We have removed this file.

File: incremental-copy-docs/README.md (hunk 1 above)
Comment (v-nikitaraut) [reply incremental-copy-docs/README.md]: Done with the changes.

File: incremental-copy-docs/README.md (hunk 2 above)
Comment (v-nikitaraut) [reply incremental-copy-docs/README.md]: Done with the changes.

File: incremental-copy-docs/README.md (hunk 3 above)
Comment (v-nikitaraut) [reply incremental-copy-docs/README.md]: Done with the changes.

**Review by erikhoward — APPROVED** [review]

**Pull request merged**